packages/synapse-react-client/src/components/SynapseTable/SynapseTableCell/SynapseTableCell.tsx

```diff
@@ -176,16 +190,25 @@ function SynapseTableCell(props: SynapseTableCellProps) {
       )
     }
     case ColumnTypeEnum.ENTITYID_LIST: {
+      const displayTextField = getDisplayTextFieldForEntityIdColumn(columnName)
       const jsonData: string[] = JSON.parse(columnValue)
       return (
         <p>
-          <EntityIdList entityIdList={jsonData} />
+          <EntityIdList
+            entityIdList={jsonData}
+            displayTextField={displayTextField}
+            showIcon={false}
+          />
         </p>
       )
     }
     case ColumnTypeEnum.USERID_LIST: {
       const jsonData: string[] = JSON.parse(columnValue)
-      return <UserIdList userIds={jsonData} />
+      return (
+        <p>
```

Comment:
To match other components, I moved this `p` outside of the `UserIdList` component